sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -41,14 +48,39 @@ object AkkaserverlessPlugin extends AutoPlugin {
       "Files that already exist they are not re-generated.")
     val temporaryUnmanagedDirectory = settingKey[File]("Directory to generate 'unmanaged' sources into")
     val temporaryUnmanagedTestDirectory = settingKey[File]("Directory to generate 'unmanaged' test sources into")
+    val excludeLogbackDependencies = settingKey[Boolean]("Exclude logback dependencies")
+    val excludeScalaTestDependencies = settingKey[Boolean]("Exclude scalatest dependencies")
   }
   object autoImport extends Keys
   import autoImport._
 
+  val LogbackVersion = "1.2.3"
+  val LogbackContribVersion = "0.1.5"
+  val logback = "ch.qos.logback" % "logback-classic" % LogbackVersion
+  val logbackJson = "ch.qos.logback.contrib" % "logback-json-classic" % LogbackContribVersion
+  val logbackJackson = "ch.qos.logback.contrib" % "logback-jackson" % LogbackContribVersion
```

Comment:
I don't think the Logback dependencies should be included by the plugin. Instead they should be included as ordinary transitive dependencies via the sdk library. I thought I had already fixed that in https://github.com/lightbend/akkaserverless-java-sdk/pull/496 ?

Reply:
Ah, did not know that.. yes is fine.

Reply:
removed these.